app/views/works/show.html.erb

```diff
@@ -40,7 +40,9 @@
 </style>
 
 <div>
-  <%= link_to("Edit", edit_work_path(@work, wizard: @work.draft? ? true : nil), class: "btn btn-primary") %>
+  <% if @work.draft?  || @work.awaiting_approval? || current_user.has_role?(:collection_admin, @work.collection) %>
+    <%= link_to("Edit", edit_work_path(@work, wizard: @work.draft? ? true : nil), class: "btn btn-primary") %>
```

Comment:
Not a change in this PR, but would plain `@work.draft?` be sufficient here? or is it important to return `nil` instead of `false`? Just trying to get familiar with this codebase.
```suggestion
    <%= link_to("Edit", edit_work_path(@work, wizard: @work.draft?), class: "btn btn-primary") %>
```

Reply:
@mccalluc interesting question.  Since I just moved the code I do not know.  That said I could try it and see if all the tests still pass, but I would not want to make it part of this PR.